.github/workflows/ci.yml

```diff
@@ -712,3 +725,166 @@ jobs:
+              echo "Not patch releasing because cannot find a matching semver tag on the branch"
+              DO_RELEASE=0
+            else
+              MAJOR=${BASH_REMATCH[1]}
+              MINOR=${BASH_REMATCH[2]}
+              PATCH=${BASH_REMATCH[3]}
+              NEXT_TAG="$MAJOR.$MINOR.$((PATCH+1))"
+              echo "NEXT_TAG is $NEXT_TAG"
+              echo "next_tag=$NEXT_TAG" >> $GITHUB_OUTPUT
+            fi
+          fi
+
+          # Check if there is anything relevant to release using GitHub API
+          # The API results include all merged pull-requests, not commits
+          # Pull-requests prefixed with MNT or DOC will not be considered relevant for releasing
+          if [[ $DO_RELEASE == "1" ]]; then
+            # Check on github release notes api if there's anything worth releasing
+            # https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#generate-release-notes-content-for-a-release
+            RESP_CODE=$(curl -w %{http_code} -s -o __response.json \
+            -X POST https://api.github.com/repos/$GITHUB_REPOSITORY/releases/generate-notes \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: Bearer ${{ github.token }}" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            -d @- << EOF
+            {
+              "tag_name": "$NEXT_TAG",
+              "target_commitish": "$LATEST_LOCAL_SHA",
+              "previous_tag_name": "$LATEST_TAG"
+            }
+          EOF
+            # ^ The EOF line above intentionally indented like it is
+            )
+            if [[ $RESP_CODE != "200" ]]; then
+              echo "Unable to fetch release notes - HTTP response code was $RESP_CODE"
+              cat __response.json
+              exit 1
+            fi
+            # Get raw body for text parsing
+            jq -r '.body' __response.json > __body.json
+            # Parse release notes one line at a time
+            IS_PRS_SECTION=0
+            HAS_THINGS_TO_RELEASE=0
+            while IFS="" read -r line || [[ -n $line ]]; do
+              echo "$line"
+              if [[ $IS_PRS_SECTION == "0" ]]; then
+                if [[ "$line" == "## What's Changed" ]]; then
+                  IS_PRS_SECTION=1
+                fi
+              else
+                if [[ "$line" =~ ^\*[[:space:]](.+?)$ ]]; then
+                  PR_TITLE=${BASH_REMATCH[1]}
+                  if ! [[ $PR_TITLE =~ ^(MNT|DOC) ]]; then
+                    HAS_THINGS_TO_RELEASE=1
+                  fi
+                else
+                  IS_PRS_SECTION=0
+                fi
+              fi
+            done < __body.json
+            if [[ $HAS_THINGS_TO_RELEASE == "0" ]]; then
+              echo "Not patch releasing because there is nothing relevant to release"
+              DO_RELEASE=0
+            fi
+          fi
+          echo "do_release output is $DO_RELEASE"
+          echo "do_release=$DO_RELEASE" >> $GITHUB_OUTPUT
```

Comment:
```suggestion
          echo "do_release output is 1"
          echo "do_release=1" >> $GITHUB_OUTPUT
```

Reply:
This doesn't make sense since I'd like to keep the `$DO_RELEASE` variable